pytorch_lightning/strategies/fully_sharded.py

```diff
@@ -172,8 +210,16 @@ def wrap_policy(*args, **kwargs):
 
         log.detail(f"{self.__class__.__name__}: exiting model_sharded_context.")
 
-    def configure_ddp(self) -> None:
-        log.detail(f"{self.__class__.__name__}: configuring DDP... (cpu_offload: [{self.cpu_offload}])")
+    def setup(self, trainer: "pl.Trainer") -> None:
+        self.accelerator.setup(trainer)
+        self.setup_optimizers(trainer)
+        self.setup_precision_plugin()
+        optimizers_to_device(self.optimizers, self.root_device)
+
```

Comment:
Seems to be missing some logic from DDP there. `_rank_0_will_call_children_scripts`.